lib/qless/server/static/css/docs.css

```diff
@@ -815,25 +815,4 @@ form.well {
     width: 1168px; /* 2px less to account for left/right borders being removed when in fixed mode */
   }
 
-}
-
-/* For proper failed job display*/
-.l-sidebyside {
-  overflow:auto;
-}
-
-.l-sidebyside > * {
-  display:inline-block;
-  *display: inline;
-  *zoom: 1;
-}
-
-.failed-job > .min-col-size {
-  min-width: 395px;
-}
-
-.failed-job .row {
-  margin-left:0;
-  margin-right:10px;
-  height: 30px;
-}
+}
```

Comment:
Can you add back the trailing newline character on this line?

Reply:
sure